dev/loadgen/configs/prod-benchmark-pvc.yaml

```diff
@@ -0,0 +1,83 @@
+## start with
+##    loadgen benchmark prod-benchmark.yaml
+
+workspaces: 1
+ideImage: eu.gcr.io/gitpod-core-dev/build/ide/code:commit-ff263e14024f00d0ed78386b4417dfa6bcd4ae2f
+waitForRunning: "600s"
+waitForStopping: "600s"
+successRate: 1
+environment:
+  - name: "THEIA_SUPERVISOR_TOKENS"
+    value: '[{"token":"foobar","host":"gitpod-staging.com","scope":["function:getWorkspace","function:getLoggedInUser","function:getPortAuthenticationToken","function:getWorkspaceOwner","function:getWorkspaceUsers","function:isWorkspaceOwner","function:controlAdmission","function:setWorkspaceTimeout","function:getWorkspaceTimeout","function:sendHeartBeat","function:getOpenPorts","function:openPort","function:closePort","function:getLayout","function:generateNewGitpodToken","function:takeSnapshot","function:storeLayout","function:stopWorkspace","resource:workspace::fa498dcc-0a84-448f-9666-79f297ad821a::get/update","resource:workspaceInstance::e0a17083-6a78-441a-9b97-ef90d6aff463::get/update/delete","resource:snapshot::*::create/get","resource:gitpodToken::*::create","resource:userStorage::*::create/get/update"],"expiryDate":"2020-12-01T07:55:12.501Z","reuse":2}]'
+workspaceClass: "g1-standard-pvc"
+featureFlags:
+  - 7
```

Comment:
I felt it is too difficult to understand what the number means. Can I ask you to put a link as a comment to make it easier to understand?

Reply:
fixed

Reply:
@sagor999 Thanks for your fixing 😍 This comment will help me.